app/models/user.rb

```diff
@@ -6,11 +6,14 @@ class User < ApplicationRecord
   validates :password, presence: true, length: { minimum: 6 }
   has_secure_password
   has_many :posts
+  has_many :comments
+
 
   # Returns true if the given token matches the digest of attribute
   def authenticated?(attribute, token)
     digest = send("#{attribute}_digest")
     return false if digest.nil?
```

Comment:
you should use `blank?` & `present?` See the example below to know the reason:

![image](https://user-images.githubusercontent.com/52812615/64394352-059bd200-d080-11e9-84a3-1e433e64b1ae.png)